CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Changelog
 
+# 13.0.0
+
+#### Breaking changes
+
+- In variables:
+  - Remove SimpleVariable and DatedVariable. Any Variable acts like old DatedVariable
```

Comment:
I don't think there is currently anything named `SimpleVariable`. 

Suggestion: 

- Merge `Variable` and `DatedVariable`
    - `Variable` can now handle formula evolution over time.